feature/subscriber-attributes/src/main/java/com/revenuecat/purchases/subscriberattributes/SubscriberAttributesManager.kt

```diff
@@ -71,10 +71,7 @@ class SubscriberAttributesManager(
                     if (didBackendGetAttributes) {
                         markAsSynced(syncingAppUserID, unsyncedAttributesForUser, attributeErrors)
                     }
-                    errorLog(
-                        "There was an error syncing subscriber attributes for " +
-                            "appUserID: $syncingAppUserID. Error: $error"
-                    )
+                    errorLog(AttributionStrings.SYNCING_ATTRIBUTES_ERROR.format(syncingAppUserID, error))
```

Comment:
We have 
success = `ATTRIBUTES_SYNC_SUCCESS`
error = `SYNCING_ATTRIBUTES_ERROR`
Maybe we should unify their naming, something like 
`SYNCING_ATTRIBUTES_ERROR` and `SYNCING_ATTRIBUTES_SUCCESS`
or
`ATTRIBUTES_SYNC_SUCCESS` and `ATTRIBUTES_SYNC_ERROR`.

Reply:
Just updated to `ATTRIBUTES_SYNC_SUCCESS` and `ATTRIBUTES_SYNC_ERROR`